src/GatherContent/HTTPClient.php

```diff
@@ -60,7 +69,80 @@ private function sendRequest($url, $additionalCurlOptions)
         $response->status = $status;
         $response->body   = $body;
 
+
+        $end = microtime(true);
+
+        print($url.': '.($end - $start).' seconds'.PHP_EOL);
```

Comment:
libraries shouldn't print - it makes it impossible for consumers to control the output.

Do we have some sort of common Logging functionality available for our PHP projects?

Reply:
I removed this. Was accidentally left in. We have some logging and if this timing information is useful we can do it higher up the stack. I agree it shouldn't be in the library itself.